src/WinGetYamlGenerator/AppInfo.cs

```diff
@@ -263,8 +263,7 @@ private bool VerifyInstallers(IList<string> errors)
         public void GenerateSuggestedAppId()
         {
             if (string.IsNullOrEmpty(publisher) || string.IsNullOrEmpty(name))
-            {
-                return;
+            { return;
```

Comment:
nit - can you put this back on separate line?

Reply:
Done, don't know how I missed that 😬